layouts/partials/article_tags.html

```diff
@@ -0,0 +1,9 @@
+{{ if isset .Params "tags" }}
+{{ if not (eq (len .Params.tags) 0) }}
+<div class="article-tags">
+  {{ range .Params.tags }}
+  <span class="badge badge-light"><a class="article-tags-link" href="{{ $.Site.BaseURL }}/tags/{{ . | urlize }}">{{ . }}</a></span>
+  {{ end }}
+</div>
+{{ end }}
+{{ end }}
```

Comment:
As of Hugo 0.65 i believe you can simply this to:
```
{{ with (.GetTerms "tags")}}
<div class="article-tags">
{{ range . }}
    <span class="badge badge-light"><a class="article-tags-link" href="{{ .Permalink }}">{{ .Name }}</a></span>
{{ end }}
</div>
{{end}}
```

Reply:
Tested it locally. The suggestion of @gwatts  works.

Reply:
Well better stated the intermediate version below works

{{ with (.GetTerms "tags")}}
<div class="article-tags">
    {{ range . }}
    <span class="badge badge-light"><a class="article-tags-link" href="{{ $.Site.BaseURL }}/tags/{{ . | urlize }}">{{ . }}</a></span>
    {{ end }}
</div>
{{end}}

Reply:
If you want to send a PR to the repo of mine used in this PR I'll happily merge it in, so its automatically added into the PR and you get the commit.
I'm in the middle of a few other things today and tomorrow, so that'll be the quickest way to go.

Reply:
Im going to mark this as resolved based on marcelbaumann's pr that I've merged into my branch.